Tests/SecureXPCTests/XPCEncoder/Scalar encoding tests.swift

```diff
@@ -0,0 +1,240 @@
+import XCTest
+@testable import SecureXPC
+
+final class XPCEncoder_ScalarEncodingTests: XCTestCase {
+	// MARK: TODO:
```

Comment:
Similar feedback as elsewhere that the framework doesn't support this today and that `XPCEncoder` would be unlikely to be the way that any of these have support added.